app/jobs/regular/remove_expert_post_properties_from_first_posts.rb

```diff
@@ -0,0 +1,58 @@
+# frozen_string_literal: true
+
+module Jobs
+  class RemoveExpertPostPropertiesFromFirstPosts < ::Jobs::Base
+    sidekiq_options queue: "low"
+
+    def fixPost(post)
+      return unless post
+      return unless post.post_number==1
```

Comment:
we try to use `if`s instead of `unless` generally, because our small brains can't wrap our heads around negatives